app/views/layouts/_header.html.erb

```diff
@@ -1,18 +1,219 @@
+            />
+            <button class="usa-button" type="submit">
+              <img
+                src="<%= image_path('images/usa-icons-bg/search--white.svg') %>"
+                class="usa-search__submit-icon"
+                alt="Search"
+              />
+            </button>
+          </form>
+        </section>
+        <% if logged_in? %>
+          <%= button_to("Logout", session_path, method: :delete, class: "usa-button font-body-3xs usa-button--outline text-no-wrap desktop:margin-left-1", type: "button") %>
+        <% else %>
+          <%= button_to("Login", new_session_path, method: :get, class: "usa-button font-body-3xs usa-button--outline text-no-wrap desktop:margin-left-1", type: "button") %>
+        <% end %>
+        <button class="usa-button font-body-2xs text-no-wrap">Create Account</button> 
+      </section>
+      <!-- mobile search and login area --> 
+      <section class="desktop:display-none order-3 flex-align-self-end">
+        <section aria-label="Search component">
+          <form class="usa-search usa-search--small margin-y-1" role="search">
+            <label class="usa-sr-only" for="search-field">Search</label>
+            <input
+              class="usa-input"
+              id="search-field"
+              type="search"
+              name="search"
+              style="width: 140px"
+            />
+            <button class="usa-button" type="submit">
+              <img
+                src="<%= image_path('images/usa-icons-bg/search--white.svg') %>"
+                class="usa-search__submit-icon"
+                alt="Search"
+              />
+            </button>
+          </form>
+        </section>
+        <% if logged_in? %>
+          <%= button_to("Logout", session_path, method: :delete, class: "usa-button font-body-3xs usa-button--outline margin-y-1 text-no-wrap desktop:margin-left-1 width-full", type: "button") %>
+        <% else %>
+          <%= button_to("Login", new_session_path, method: :get, class: "usa-button font-body-3xs usa-button--outline margin-y-1 text-no-wrap desktop:margin-left-1 width-full", type: "button") %>
+        <% end %>
+        <button class="usa-button font-body-2xs text-no-wrap width-full margin-y-1">Create Account</button> 
+      </section>
     </nav>
   </div>
 </header>
+<div class="bg-primary-darker w-full display-flex flex-justify-center desktop:flex-justify-end padding-1 font-body-2xs ">
+  <div class="display-flex flex-align-center margin-x-1">
+    <%= image_tag('images/usa-icons/grid_view.svg', class: "usa-icon--size-3 icon-white margin-top-1", alt: "Dashboard") %>
+    <a href="javascript:void(0);" class="text-white margin-top-1">
+      Dashboard
+    </a>
+  </div>
+  <div class="display-flex flex-align-center margin-x-1">
+    <%= image_tag('images/usa-icons/folder_open.svg', class: "usa-icon--size-3 icon-white margin-top-1", alt: "Submissions") %>    
+    <a href="javascript:void(0);" class="usa-nav-link text-white margin-top-1">
```

Comment:
this will probably be dynamic based on the user role but could be the same route anyways
```suggestion
    <a href="/submissions" class="usa-nav-link text-white margin-top-1">
```